infrastructure/README.md

```diff
@@ -14,3 +14,7 @@ A repository of Infrastructure solutions.
 
 ### [cloudconnect](cloudconnect/)
   CloudConnect is a package that automates the setup of dual VPN tunnels between AWS and GCP. While this connection is **NOT** meant for high throughput, low latency connections (1Gbps+), it can certainly support basic management and configuration traffic. Currently it supports the creation of both _static-routes_ and _bgp_ connections.
+
+### [gsuite-exporter](gsuite-exporter/)
```

Comment:
Move to top-level README in new structure.

Reply:
Done.